triggers/service/src/main/scala/com/digitalasset/daml/lf/engine/trigger/package.scala

```diff
@@ -44,5 +45,8 @@ package trigger {
       triggerAccessToken: Option[AccessToken],
       triggerRefreshToken: Option[RefreshToken],
       triggerReadAs: Set[Party],
-  )
+  ) {
+    private[trigger] def withLoggingContext[T]: (LoggingContextOf[Trigger] => T) => T =
```

Comment:
you’re not incuding the uuid here. Any reason for that? I think we should include it.

Reply:
2be7868e210289935cb8b9a53aec586b45000d74